cluster/examples/kubernetes/ceph/operator-openshift.yaml

```diff
@@ -100,28 +99,65 @@ metadata:
   # should be in the namespace of the operator
   namespace: rook-ceph
 data:
-  # # (Optional) Ceph Provisioner NodeAffinity.
+  # Enable the default version of the CSI CephFS driver. To start another version of the CSI driver, see image properties below.
+  ROOK_CSI_ENABLE_CEPHFS: "true"
```

Comment:
Please also make the changes to use the configmap in the integration tests. The settings would be found in ceph_manifests.go

Reply:
Done.